worlds/khrecom/docs/en_Kingdom Hearts RE Chain of Memories.md

```diff
@@ -0,0 +1,42 @@
+# Kingdom Hearts RE Chain of Memories (PC)
+
+## Where is the settings page?
+
+The [player settings page for this game](../player-settings) contains most of the options you need to 
+configure and export a config file.
```

Comment:
```suggestion
## Where is the options page?

The [player options page for this game](../player-options) contains most of the options you need to 
configure and export a config file.
```
Settings was changed to Options a while back

Reply:
Forgot to fix this when I did it for KH1, fixed!